test_rclcpp/test/test_executor.cpp

```diff
@@ -28,12 +28,15 @@
 TEST(CLASSNAME(test_executor, RMW_IMPLEMENTATION), recursive_spin_call) {
   rclcpp::executors::SingleThreadedExecutor executor;
   auto node = rclcpp::Node::make_shared("recursive_spin_call");
-  auto timer = node->create_wall_timer(0_s, [&executor]() {
+  auto timer = node->create_wall_timer(0_s,
```

Comment:
Please wrap either all arguments or none. In this case add a newline after the opening parenthesis.